### PR TITLE
Add cli utilities.

### PR DESCRIPTION
**Pull request by jmrozanec**
Changes proposed in this pull request:

* provide a multiflow.sh script, which helps to setup dev environment and run tests

* Fixes # [Optional]

---

#### Implementation
- [x] Implementation is correct (it performs its intended function).
- [x] Code is consistent with the framework.
- [x] Code is properly documented.
- [x] PR description covers ALL the changes performed.
- [x] Files changed (update, add, delete) are in the PR's scope (no extra files are included).

#### Tests
- [x] New functionality is tested.
- [x] Tests are created for the new functionality or existing tests are updated accordingly.
- [x] ALL tests pass with no errors.
- [x] CI/CD pipelines run with no errors.
- [x] Test Coverage is maintained (coverage may drop by no more than 0.2%).

### CODE REVIEW

**Comment by jmrozanec** [comment]
@garawalid here the promised script for plain code 😄

File: multiflow.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+set -e # Ensure we fail if an error occurs
+set -o pipefail # Ensure we fail if something fails at pipeline level
+
+######################## GET SCRIPT LOCATION - START - ##############################
+SCRIPT_LOCATION="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+######################## GET SCRIPT LOCATION - END - ################################
+
+PROFILE="${PROFILE:-LOCAL}"
```
Comment (garawalid) [inline multiflow.sh:9]: Is it necessary to specify the profile for the moment ?

**Review by garawalid — COMMENTED** [review]
Thanks for your work and this PR! I left you some comments

File: multiflow.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
```
Comment (garawalid) [inline multiflow.sh:1]: I'd move `multiflow.sh` to `dev` directory and rename it **skmultiflow.sh**

**Pull request closed without merging**